### DISCUSSION TITLE
Can anyone in the community share any experience "integrating" Quarto with a wiki tool?

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#8529] [Q&A] [answered]
### Description

I am working with a team that wants to use a wiki for internal documentation.  We are looking for a wiki tool.  I see that Quarto has support to generate Confluence pages.  I was wondering if anyone has some experience to share generating wiki pages for an open source wiki.

**Comment by mcanouil** [comment]
I once try GitHub Wiki with Quarto but Wiki has too many constraints so switch to full Quarto website and GitHub Pages.

**Comment by fredguth** [comment]
The only wiki feature that I really miss if I just use a Quarto website are automatic reference to pages that link to current page.

**Reply by mcanouil** [reply, marked as answer]
backlinks?

- https://github.com/quarto-dev/quarto-cli/issues/1585

**Reply by fredguth** [reply]
Exactly, thanks for the reference.
